src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/_factory.py

```diff
@@ -83,9 +90,10 @@ def queue_data_service_factory(kwargs):
         connection_string=kwargs.pop('connection_string', None),
         sas_token=kwargs.pop('sas_token', None))
 
+
 def cloud_storage_account_service_factory(kwargs):
     account_name = kwargs.pop('account_name', None)
     account_key = kwargs.pop('account_key', None)
     sas_token = kwargs.pop('sas_token', None)
-    connection_string = kwargs.pop('connection_string', None) # pylint: disable=unused-variable
+    kwargs.pop('connection_string', None)  # pylint: disable=unused-variable
```

Comment:
But the original code did nothing to the `connection_string` variable and it is marked to disable the `unused-variable` check, which I don't understand the reason why. Given that the tests all pass, anything specifically I should change in addition to what already is in this pull request?